Failure in no_skip.py:

  The CHECK on line 6 wants:
    This should never be compared

  which failed to match line stdout:1:
    Oh, but it is!

  Context:
    Oh, but it is! <= does not match CHECK on line 6: This should never be compared

  when running command:
    /usr/bin/python3 no_skip.py
